### ISSUE TITLE
sysutils/smart: Syntax Error in diag_smart.php

### ISSUE THREAD

**Issue by fabianfrz** [opnsense/plugins#714]
@fichtner just because we are currently talking about it

```
find (Path)/plugins/sysutils/smart/src  ! -name "*.xml" ! -name "*.xml.sample" ! -name "*.eot"  ! -name "*.svg" ! -name "*.woff" ! -name "*.woff2"  ! -name "*.otf" ! -name "*.png" ! -name "*.js"  ! -name "*.scss" ! -name "*.py" ! -name "*.ttf"  ! -name "*.tgz" ! -name "*.xml.dist" ! -name "*.sh"  -type f -print0 | xargs -0 -n1 php -l
PHP Parse error:  syntax error, unexpected '}', expecting end of file in /(Path)/plugins/sysutils/smart/src/www/diag_smart.php on line 307
Errors parsing (Path)/plugins/sysutils/smart/src/www/diag_smart.php
xargs: php: mit Rückgabewert 255 beendet, Abbruch.
*** Error code 124
```

**Comment by fichtner** [comment]
Not on PHP 7.1 with macOS nor FreeBSD/OPNsense

**Comment by fabianfrz** [comment]
mine is on Linux: PHP 7.2.6 (cli) (built: May 26 2018 07:45:18) ( NTS )

**Comment by fichtner** [comment]
I'll fix when we have a test for 7.2, currently planned for later in 18.7.x. Otherwise you can grab this ticket.

**Comment by fichtner** [comment]
#715
